pgxn/neon/libpagestore.c

```diff
@@ -421,12 +421,14 @@ pageserver_connect(shardno_t shard_no, int elevel)
 		 */
 		keywords[0] = "dbname";
 		values[0] = connstr;
-		n_pgsql_params = 1;
+		keywords[1] = "keepalives";
+		values[1] = "1";
```

Comment:
The default is `1` already: https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-KEEPALIVES.